tools/tracegen/Port.cpp

```diff
@@ -30,8 +30,10 @@
 #define J9FILE_UNC_EXTENDED_LENGTH_PREFIX (L"\\\\?\\")
 #else /* defined(WIN32) */
 #include <sys/types.h>
-#if !defined(OMRZTPF)
+#if defined(AIXPPC)
 #include <sys/statvfs.h>
+#elif (defined(LINUX) && !defined(J9ZTPF)) || defined(OSX)
```

Comment:
That code has not really changed since the initial commit.  The zptf changes recently would not have changed anything for OSX.  I am wary of changes we do not fully understand